package.json

```diff
@@ -0,0 +1,26 @@
+{
+	"name": "@microsoft/atlas-design",
+	"version": "1.0.0",
+	"description": "The Atlas Design System used by Microsoft's Developer Relations platform.",
+	"scripts": {
+		"test": "echo \"Error: no test specified\" && exit 1"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/microsoft/atlas-design"
+	},
+	"workspaces": {
+		"packages": [
+			"css"
+		]
+	},
+	"keywords": [
+		"microsoft",
+		"atlas",
+		"design-system",
+		"css",
+		"scss"
+	],
+	"author": "Microsoft Corporation",
+	"license": "MIT"
```

Comment:
Thinking this PR should include a `LICENCE` file. Alternatively, there's a method to add one via the UI:
https://docs.github.com/en/github/building-a-strong-community/adding-a-license-to-a-repository

Reply:
Microsoft open source took care of that for us: https://github.com/microsoft/atlas-design/blob/main/LICENSE